.github/workflows/ci.yml

```diff
@@ -407,9 +410,8 @@ jobs:
       - uses: actions/download-artifact@v3
         with:
           name: babel-artifact
-      - name: Generate runtime helpers
```

Comment:
This step is removed because the runtime helpers are stored in the artifacts, too.